headers/data/arm9.h

```diff
@@ -17,6 +17,21 @@ extern uint32_t MONSTER_ID_LIMIT;
 extern uint32_t MAX_RECRUITABLE_TEAM_MEMBERS;
 extern int16_t NATURAL_LOG_VALUE_TABLE[2048];
 extern undefined CART_REMOVED_IMG_DATA[0]; // Length differs between versions
+extern char STRING_DEBUG_EMPTY[0];
+extern char STRING_DEBUG_FORMAT_LINE_FILE[26];
+extern char STRING_DEBUG_NO_PROG_POS[20];
+extern char STRING_DEBUG_SPACED_PRINT[9];
+extern char STRING_DEBUG_FATAL[18];
+extern char STRING_DEBUG_NEWLINE[1];
+extern char STRING_DEBUG_LOG_NULL[7];
+extern char STRING_DEBUG_STRING_NEWLINE[3];
+extern char STRING_EFFECT_EFFECT_BIN[17];
+extern char STRING_MONSTER_MONSTER_BIN[19];
+extern char STRING_BALANCE_M_LEVEL_BIN[19];
+extern char STRING_DUNGEON_DUNGEON_BIN[19];
+extern char STRING_MONSTER_M_ATTACK_BIN[20];
+extern char STRING_MONSTER_M_GROUND_BIN[20];
+extern char STRING_FILE_DIRECTORY_INIT[36];
```

Comment:
These buffers should include the space needed for the null terminator

Reply:
I wasn't thinking carefully, sorry. I think these ought to match the underlying data symbol length, which I assume includes not just the null terminator but also any padding in the binary. I updated the lengths, lmk if you object.

Reply:
Fine by me.